wagtail/contrib/redirects/views.py

```diff
@@ -126,3 +133,183 @@ def add(request):
     return TemplateResponse(request, "wagtailredirects/add.html", {
         'form': form,
     })
+
+
+@permission_checker.require_any("add")
+def start_import(request):
+    from_encoding = "utf-8"
+    query_string = request.GET.get('q', "")
+
+    if not request.POST:
+        SUPPORTED_FORMATS = ["CSV", "TSV", "XLS", "XLSX"]
```

Comment:
Another suggestion - as this is the canonical source for what formats the user can upload, it might be best to move this to a class property or static method if possible. 

As this is a function view, maybe on the redirect model or just a global in this file. 

Just somewhere central, I think it will come in handy having this outside of a branch within the view.

Reply:
Agree! I'll make it a module property.